sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -217,9 +217,60 @@
 #--------------------------------------------------------------------------------------------------
 # LOGGING
 
-# Level of logs. Supported values are INFO(default), DEBUG and TRACE (DEBUG + SQL + ES requests)
+# SonarQube produces logs in 4 logs files located in the same directory (see property sonar.path.logs below),
+# one per JVM: App (main process), Web Server (aka. Web), Compute Engine (aka. CE) and Elastic Search (aka. ES)
+# All 4 files have logs in the same format:
+#           1           2    3           4                       5                                                   6
+# |-----------------| |---| |-|--------------------||------------------------------| |------------------------------------------------------------------------------------------------------------------------------|
+# 2016.11.16 16:47:00 INFO  ce[AVht0dNXFcyiYejytc3m][o.s.s.c.t.CeWorkerCallableImpl] Executed task | project=org.sonarqube:example-java-maven | type=REPORT | id=AVht0dNXFcyiYejytc3m | submitter=admin | time=1699ms
+#
+# 1: timestamp. Format is YYYY.MM.DD HH:MM:SS
+#    YYYY: year on 4 digits
+#    MM: month on 2 digits
+#    DD: day on 2 digits
+#    HH: hour of day on 2 digits in 24 hours format
+#    MM: minutes on 2 digits
+#    SS: seconds on 2 digits
+# 2: log level. Possible values are (in order of descending criticality): ERROR, WARN, INFO, DEBUG and TRACE
+# 3: process identifier. Possible values are: app (main process), web (Web Server), ce (Compute Engine) and es (Elastic Search)
+# 4: thread identifier. Can be empty.
+#    In the Web Server, if present, it will be the HTTP request identifier.
+#    In the Compute Engine, if present, it will be the task UUID.
+# 5: logger name. Usually a class canonical name.
+#    Package names are truncated to keep the whole field to 20 characters max
+# 6: log payload. Content of this field does not follow any specific format, can vary in length and include line returns.
+#    Some logs, however, will follow the convention to provide data in payload in the format " | key=value"
+#    Especially, log of profiled pieces of code will end with " | time=XXXXms".
+
+# Global level of logs (applies to App, Web and CE processes).
+# Supported values are INFO (default), DEBUG and TRACE
 #sonar.log.level=INFO
 
+# Level of logs of each process can be controlled individually with their respective properties.
+# When specified, they overwrite the level defined at global level.
+# Supported values are INFO, DEBUG and TRACE
+#sonar.log.level.app=INFO
+#sonar.log.level.web=INFO
+#sonar.log.level.ce=INFO
+#sonar.log.level.es=INFO
+
+# Sql logs in the Web and CE processes can be individually controlled with their respective properties
+# Supported values are INFO, DEBUG and TRACE
+# Currently, there is Sql logs only at TRACE level
+#sonar.log.level.web.sql=TRACE
```

Comment:
which inconsistency? in the value of the commented property or the absence of default value?